content/en/docs/Responses/transcodeDecisionResponse.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "transcodeDecisionResponse"
+linkTitle: "transcodeDecisionResponse [OS]"
+opensubsonic:
+- Extension
+description: >
+  The response from the getTranscodeDecision endpoint.
+---
+
+{{< tabpane persist=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic" lang="json">}}
+{
+  "subsonic-response": {
+    "status": "ok",
+    "version": "1.16.1",
+    "type": "AwesomeServerName",
+    "serverVersion": "0.1.3 (tag)",
+    "openSubsonic": true,
+    "transcodeDecision": {
+      "canDirectPlay": false,
+      "canTranscode": true,
+      "transcodeReason": ["AudioCodecNotSupported"],
+      "errorReason": "",
+      "transcodeUrl": "/rest/getTranscodeStream.view?trackID=123&offset=0&parameters=...",
+      "sourceStream": {
+        "protocol": "http",
+        "container": "flac",
+        "codec": "flac",
+        "audioChannels": 6,
+        "audioBitrate": 3000000,
+        "audioProfile": "",
+        "audioSamplerate": 96000,
+        "audioBitdepth": 24
+      },
+      "transcodeStream": {
+        "protocol": "hls",
+        "container": "mp4",
+        "codec": "aac",
+        "audioChannels": 2,
+        "audioBitrate": 256000,
+        "audioProfile": "xHE-AAC",
+        "audioSamplerate": 48000,
+        "audioBitdepth": 16
+      }
+    }
+  }
+}
+{{< /tab >}}
+{{< /tabpane >}}
+
+| Field | Type | Req. | OpenS. | Details |
+| --- | --- | --- | --- | --- |
+| `canDirectPlay` | boolean | **Yes** | **Yes** | Whether the media can be played directly by the client. |
+| `canTranscode` | boolean | **Yes** | **Yes** | Whether the media can be transcoded by the server. |
+| `transcodeReason` | string[] | No | **Yes** | An array of reasons why transcoding is necessary. Possible values are: `AudioCodecNotSupported`, `AudioBitrateNotSupported`, `AudioChannelsNotSupported`, `AudioSampleRateNotSupported`, `ContainerNotSupported`, `ProtocolNotSupported`. |
```

Comment:
Would it be worthwhile making this an enum instead?

Reply:
It would be in the OpenAPI but I don't think we ever defined enums in the API no ?